test/e2e/logs_test.go

```diff
@@ -30,6 +30,10 @@ func (tc *testContext) testNodeLogs(t *testing.T) {
 		"csi-proxy/csi-proxy.log",
 	}
 	nodeArtifacts := filepath.Join(os.Getenv("ARTIFACT_DIR"), "nodes")
+	err := retrieveWindowsNodeObjects(nodeArtifacts)
+	if err != nil {
+		log.Printf("error retrieving windows node objects: %s", err)
+	}
```

Comment:
Don't see why this should be in the test for the logs endpoint.
I suggest moving it up a level, [to the end of the creation test suite](https://github.com/openshift/windows-machine-config-operator/blob/1b88ba5a16ef103c607f3cf3d0bb5cf4c95baf6c/test/e2e/create_test.go#L38)

Reply:
@sebsoto, could you share more about the benefit (why) of moving this to the creation test suite?

IMO, here is where the `nodes`  path is defined, and I don't feel we need to duplicate it.

Lastly, I'm proposing a structure similar to the must-gather where the `nodes.json` containing the Windows node objects sits next to the node logs. For example:

```
nodes/nodes.json
nodes/e2e-wm-k7msf/
nodes/winhost-n6fse/
```

Reply:
@sebsoto, could you share more about the benefit (why) of moving this to the creation test suite?

IMO, here is where the `nodes`  path is defined, and I don't feel we need to duplicate it.

Lastly, I'm proposing a structure similar to the must-gather where the `nodes.json` containing the Windows node objects sits next to the node logs. For example:

```
nodes/nodes.json
nodes/e2e-wm-k7msf/
nodes/winhost-n6fse/
```

Reply:
@sebsoto  Addressed in https://github.com/openshift/windows-machine-config-operator/pull/2640/commits/585efccf5075e05e448338fff2828e38333511c4